dashboard/lib/service/google_authentication.dart

```diff
@@ -73,7 +73,6 @@ class GoogleSignInService extends ChangeNotifier {
     }
 
     final String idToken = await user?.authentication?.then((GoogleSignInAuthentication key) => key.idToken);
-    assert(idToken != null && idToken.isNotEmpty);
```

Comment:
Why was this removed?

Reply:
just based on my personal testing in scenarios like, if I click rerun and close the pop up sign in dialogue, then the system will throw an exception due to this implementation, and a generic error information will be shown. I removed it so that I can control sign in related error message display in other functions outside google auth. Maybe there is a better way to do it instead of removing it?

Reply:
If you're running this on web, try using `flutter run -d chrome --web-port=8080`. The Google Sign In app will only run on that port for local runs

Reply:
umm yeah i did that and on https://flutter-dashboard.appspot.com/#/build, if i click on a box for rerun and close the sign in dialogue, it shows generic error (which from my understanding is caused by exception thrown). On local web I tested by printing out the exception. I think it is rare that people would close a sign in dialogue after it pops up, umm are you able to reproduce the scenario i described?

Reply:
This looks to be caused by the fake data (we could talk a lot more about this but tl;dr we can support prod data only on the desktop platforms, but you can't sign in as the plugin doesn't have desktop support, and web we can support prod sign in, but can't support prod data due to CORS)